### PR TITLE
zh: cdc document

### PR DESCRIPTION
**Pull request by weekface**
### What is changed, added or deleted? (Required)

add cdc documents, close: https://github.com/pingcap/tidb-operator/issues/2492.

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

File: zh/configure-cluster-using-tidbcluster.md [hunk 1]
```diff
@@ -100,4 +100,22 @@ spec:
...
+
+```yaml
+apiVersion: pingcap.com/v1alpha1
+kind: TidbCluster
+metadata:
+  name: basic
+spec:
+  ...
+  ticdc:
+    config:
```
Comment (DanielZhangQD) [inline zh/configure-cluster-using-tidbcluster.md:117]: Is there any default configuration file that users can refer to for all of the configurations? If yes, we'd better add a link here.
There is no such file for TiFlash but there is default config file for pd, tidb and tikv.

File: zh/deploy-ticdc.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+* TiDB Operator [部署](deploy-tidb-operator.md)完成。
+
+## 全新部署 TiDB 集群同时部署 TiCDC
+
+参考 [在标准 Kubernetes 上部署 TiDB 集群](deploy-on-general-kubernetes.md)进行部署。
+
+## 在现有 TiDB 集群上新增 TiCDC 组件
+
+编辑 TidbCluster Custom Resource：
```
Comment (ran-huang) [inline zh/deploy-ticdc.md:21]: Please add `1, 2, 3, ...` to mark these steps.

File: zh/deploy-ticdc.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+部署完成后，通过 `kubectl exec` 进入任意一个 TiCDC Pod 进行操作。
+
+{{< copyable "shell-regular" >}}
+
+```shell
+kubectl exec -it ${pod_name} -n ${namespace} sh
+```
+
+然后通过 `cdc cli` 进行[管理集群和同步任务](https://pingcap.com/docs-cn/stable/ticdc/manage-ticdc/)。
```
Comment (ran-huang) [inline zh/deploy-ticdc.md:46]: Is there any result or ouput that users can expect after this step?

File: zh/configure-cluster-using-tidbcluster.md (hunk 1 above)
Comment (weekface) [reply zh/configure-cluster-using-tidbcluster.md:117]: CDC 没有配置文件，这些都是启动参数。

Comment (weekface) [reply zh/configure-cluster-using-tidbcluster.md:117]: 链接已加

File: zh/configure-cluster-using-tidbcluster.md [hunk 4]
```diff
@@ -100,4 +100,22 @@ spec:
         logger:
           count: 5
           level: information
-```
\ No newline at end of file
+```
+
+## 配置 TiCDC 启动参数
+
+你可以通过 TidbCluster Custom Resource 的 `spec.ticdc.config` 来配置 TiCDC 启动参数，以下是一个例子。获取所有可以配置的 CDC 启动参数，请参考 [CDC 启动参数文档](https://pingcap.com/docs-cn/stable/ticdc/deploy-ticdc/#手动在原有-tidb-集群上新增-ticdc-组件)
```
Comment (ran-huang) [inline zh/configure-cluster-using-tidbcluster.md:107]:
```suggestion
你可以通过 TidbCluster Custom Resource 的 `spec.ticdc.config` 来配置 TiCDC 启动参数，以下是一个例子。获取所有可以配置的 CDC 启动参数，请参考 [CDC 启动参数文档](https://pingcap.com/docs-cn/stable/ticdc/deploy-ticdc/#手动在原有-tidb-集群上新增-ticdc-组件)。
```

**Pull request merged**